bundles/org.openhab.binding.vigicrues/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -59,45 +59,72 @@
 		<label>Observation Time</label>
 		<description>Observation date and time</description>
 		<category>time</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Timestamp</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="height">
 		<item-type>Number:Length</item-type>
 		<label>Height</label>
 		<description>Water level in the river</description>
+		<category>flowpipe</category>
```

Comment:
I don't think this is an official category (with an icon in the classic iconset).

Reply:
Yes, it is and shows a nice blue arrow

Reply:
Verified ;)